conf/drupal/config/field.field.node.org_page.field_bg_wide.yml

```
@@ -11,9 +11,9 @@ id: node.org_page.field_bg_wide
 field_name: field_bg_wide
 entity_type: node
 bundle: org_page
-label: 'Banner Image (Desktop)'
-description: "Required. Choose a high quality image that relates to the page's overarching theme, and does not need text explanation. No alternative text is provided for screen reader users. Aspect ratio of 4:1. Minimum of 1600 x 400 pixels. Images using a different ratio will be cropped; smaller images will be distorted. If you need images, <a target=\"_blank\" href=\"https://massgovdigital.gitbook.io/knowledge-base/documents-images-media/images#image-library\">you can use our Image Library</a>.\r\n<br>\r\n<br>\r\nA white crosshair shows the focal point of the image that should remain in view (but not necessarily in the exact center) as the image scales for different browser sizes. Click on any part of the image to change the focal point.\r\n<br>\r\n"
-required: false
+label: 'Banner Image'
+description: "Required. Choose a high quality image that relates to the page's overarching theme, and does not need text explanation. No alternative text is provided for screen reader users. Aspect ratio of 2:1. Minimum of 800 x 400 pixels. Images using a different ratio will be cropped; smaller images will be distorted. If you need images, <a target=\"_blank\" href=\"https://massgovdigital.gitbook.io/knowledge-base/documents-images-media/images#image-library\">you can use our Image Library</a>.\r\n<br>\r\n<br>\r\nA white crosshair shows the focal point of the image that should remain in view (but not necessarily in the exact center) as the image scales for different browser sizes. Click on any part of the image to change the focal point.\r\n<br>\r\n"
```

```suggestion
description: "Optional. Choose a high quality image that relates to the page's overarching theme, and does not need text explanation. No alternative text is provided for screen reader users. Aspect ratio of 2:1. Minimum of 800 x 400 pixels. Images using a different ratio will be cropped; smaller images will be distorted. If you need images, <a target=\"_blank\" href=\"https://massgovdigital.gitbook.io/knowledge-base/documents-images-media/images#image-library\">you can use our Image Library</a>.\r\n<br>\r\n<br>\r\nA white crosshair shows the focal point of the image that should remain in view (but not necessarily in the exact center) as the image scales for different browser sizes. Click on any part of the image to change the focal point.\r\n<br>\r\n"
```
I think we should make all background images optional

---

Done for the org page.